release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2032,6 +2032,20 @@ For more information, see xref:../networking/metallb/metallb-upgrading-operator.
 
 * If the pruner fails, the Image Registry Operator is degraded until the pruner successfully runs. By default, the pruner runs once a day. As a workaround, you can give it more opportunities to run successfully by configuring it to run more often. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1990125[*BZ#1990125*])
 
+* When running {product-title} on bare-metal IBM Power, there is a known issue that the Petitboot bootloader is unable to populate boot configurations for some {op-system} live images. In such cases, after booting nodes with PXE to install {op-system}, the expected live image disk configuration might not be visible.
++
+As a workaround, you can boot manually with `kexec` from the Petitboot shell.
++
+Identify the disk that holds the live image, in this example `nvme0n1p3`, and run the following commands:
++
+[source,bash]
+----
+# cd /var/petitboot/mnt/dev/nvme0n1p3/ostree/rhcos-*/
+# kexec -l vmlinuz-*.ppc64le -i initramfs-*.img -c "ignition.firstboot rd.neednet=1 ip=dhcp $(grep options /var/petitboot/mnt/dev/nvme0n1p3/loader/entries/ostree-1-rhcos.conf | sed 's,^options ,,')" && kexec -e
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2107674[BZ#2107674])
```

Comment:
```suggestion
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2107674[*BZ#2107674]*)
```
Apologies @alishaIBM. I didn't catch this first time through. The BZ link needs to be bold.

Reply:
Thanks @mjpytlak for the correction. 
As discussed [over slack conversation](https://coreos.slack.com/archives/C025XCRRH28/p1659020167361189?thread_ts=1659005775.250879&cid=C025XCRRH28) this change will be handled by the RH docs team.

Marking the comment as resolved.